pulp_python/app/viewsets.py

```diff
@@ -624,3 +651,27 @@ def create(self, request):
             kwargs={"repository_version_pk": str(repository_version.pk)},
         )
         return core_viewsets.OperationPostponedResponse(result, request)
+
+
+async def get_repo_version_content(repo_version_pk: str):
+    """
+    Retrieve Python package content from a repository version for vulnerability scanning.
+    """
+    repo_version = await sync_to_async(RepositoryVersion.objects.get)(pk=repo_version_pk)
+    content_units = python_models.PythonPackageContent.objects.filter(pk__in=repo_version.content)
```

Comment:
```suggestion
    repo_version = await RepositoryVersion.objects.aget(pk=repo_version_pk)
    content_units = python_models.PythonPackageContent.objects.filter(pk__in=repo_version.content).values("name", "version")
```

Reply:
For the 
```
content_units = python_models.PythonPackageContent.objects.filter(pk__in=repo_version.content).values("name", "version")
```
I think this will not work because we also need the `content` objects found:
https://github.com/pulp/pulp_python/pull/1013/files#diff-38cc3b67ebd1f8ea44002fc7897552b1674435d85771f6ab27a0c86e383d00aeR666
https://github.com/pulp/pulpcore/blob/d9651981e20ac5b2dc1997f181204924f5c36cce/pulpcore/app/tasks/vulnerability_report.py#L133-L134